Scuti/Scripts/UI/Store/TopBarView.cs

```diff
@@ -33,7 +33,11 @@ private void Awake()
             // Instance banners
             for (int i = 0; i < amountBanners - 1; i++)
             {
-                additionalBanners[i] = Instantiate(Banner, contentBanners.transform);
+                BannerWidget banner = Instantiate(Banner, contentBanners.transform);
+                banner.gameObject.name = "Banner -"+ (int)(i+1);
+                banner.SetIndex(i+1);
```

Comment:
@Jpablogs8 I don't think this is going to work.  We'll want to separate them more. Otherwise they are going to see the same banner a lot.  Maybe do i+amountBanners?

Reply:
i agree